tests/generic_config_updater/field_operation_validator_test.py

```diff
@@ -188,6 +188,92 @@ def test_validate_field_operation_illegal__rm_loopback0(self):
         config_wrapper = gu_common.ConfigWrapper()
         self.assertRaises(gu_common.IllegalPatchOperationError, config_wrapper.validate_field_operation, old_config, target_config)
 
+    def test_validate_field_operation_illegal__dataacl_table_type_and_rule(self):
```

Comment:
Suggest unit test name should contain:
1. expectation (success or fail)
2. why the expectation (failure reason)

Hence, maybe consider a name like `test_validate_field_operation_fail_illegal_type`, `test_validate_field_operation_fail_type_changed`, etc.

Reply:
Here is the convention in this file. 
legal indicates success, illegal indicates fail.
Reason is the description after __

Reply:
Got it. Thanks.